RELEASE_NOTES.md

```diff
@@ -1,5 +1,8 @@
 Release Notes
 =============
+## 1.6.10
```

Comment:
You can use ## vNext and the farmer team will update the version once its actually released

Reply:
Thanks for the tip, that should stop merge-conflicts too.